From 6f5415d1a446c8630df955d587963869a54d0bfc Mon Sep 17 00:00:00 2001
From: Shamim Ali <shamima@ami.com>
Date: Sat, 12 Nov 2022 20:52:15 +0530
Subject: [PATCH] Add SEl policy chacher

Added a SelPolicy property which can be configured as
Linear or Circular to chose SEL logging

Signed-off-by: Shamim Ali <shamima@ami.com>
---
 include/defaults.hpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/defaults.hpp b/include/defaults.hpp
index d8bda87..1a3d250 100644
--- a/include/defaults.hpp
+++ b/include/defaults.hpp
@@ -356,6 +356,7 @@ inline void loadSettings(sdbusplus::asio::object_server &objectServer,
                                      "xyz.openbmc_project.Logging.Settings");
 
     setting->addProperty("QuiesceOnHwError", false);
+    setting->addProperty("SelPolicy", "xyz.openbmc_project.Logging.Settings.Policy.Circular");
 
     setting = &settings.emplace_back(objectServer,
                                      "/xyz/openbmc_project/pfr/last_events",
-- 
2.25.1

